client/my-sites/invites/invite-accept-logged-out/index.jsx

```diff
@@ -60,11 +60,40 @@ export default React.createClass( {
 		)
 	},
 
-	footerLink() {
+	subscribeUserByEmailOnly() {
+		this.setState( { submitting: true } );
+		acceptInvite(
+			this.props.invite,
+			( error ) => {
+				if ( error ) {
+					this.setState( { error } );
+				} else {
+					window.location = 'https://subscribe.wordpress.com?update=activate&email=' + encodeURIComponent( this.props.invite.meta.sent_to ) + '&key=' + this.props.invite.authKey;
+				}
+			}
+		);
+	},
+
+	renderFooterLink() {
 		let logInUrl = config( 'login_url' ) + '?redirect_to=' + encodeURIComponent( window.location.href );
 		return (
-			<a href={ logInUrl } className="logged-out-form__link">
-				{ this.translate( 'Already have a WordPress.com account? Log in now.' ) }
+			<div>
+				<a href={ logInUrl } className="logged-out-form__link">
+					{ this.translate( 'Already have a WordPress.com account? Log in now.' ) }
+				</a>
+				{ this.renderEmailOnlySubscriptionLink() }
+			</div>
+		);
+	},
+
+	renderEmailOnlySubscriptionLink() {
+		if ( this.props.invite.meta.role !== 'follower' || ! this.props.invite.activationKey ) {
```

Comment:
At first glance, I thought this could prevent a potential issue with `viewers` on private WPCOM sites since `viewers` are also labeled as `followers` in the code.

But, since we control the invitation email, and since we don't need an activation key for viewers, I don't anticipate this being an issue.